docs/src/components/Utils/ComponentInfo.component.tsx

```diff
@@ -2,14 +2,28 @@
 import * as React from 'react';
 // VENDOR
 import styled from '@xstyled/styled-components';
+import { th } from '@xstyled/system';
 import { Typography } from '@retailmenot/anchor';
 // COMPONENTS
-import { sections, SiteLink } from '../Utils';
+import { COMPONENTS, sections, SiteLink } from '../Utils';
 
 const StyledArticle = styled('article')`
     padding: 1rem;
 `;
 
+const TitleLink = styled('a')`
```

Comment:
Some of this could be props added to the `Typography` component. Why not add it that way?

Reply:
Originally  I did that, but changing the `scale` prop on `Typography` along with `a` as the `tag` doesn't change its size or line-height, so I went this route. You pointing it out here reminds me that I wanted to make a ticket to look into this as I wasn't sure if it was a bug or working-as-intended like the header tags.